Dockerfile

```diff
@@ -281,7 +316,17 @@ RUN git init . && git remote add origin "https://github.com/opencontainers/runc.
 # project first, and update both after that is merged. When updating RUNC_VERSION,
 # consider updating runc in vendor.mod accordingly.
 ARG RUNC_VERSION=v1.1.7
-RUN git fetch -q --depth 1 origin "${RUNC_VERSION}" +refs/tags/*:refs/tags/* && git checkout -q FETCH_HEAD
+ARG TARGETARCH
+RUN <<EOT
+  set -e
+  if [ "$TARGETARCH" = "riscv64" ]; then
+    # riscv64 is not supported by latest runc 1.1.7, so use the release 1.1
+    # branch that brings support for riscv64 (https://github.com/opencontainers/runc/pull/3905).
+    RUNC_VERSION=release-1.1
+  fi
+  git fetch -q --depth 1 origin "${RUNC_VERSION}" +refs/tags/*:refs/tags/*
+  git checkout -q FETCH_HEAD
+EOT
```

Comment:
@AkihiroSuda Made this change while waiting for next patch release (related to https://github.com/opencontainers/runc/pull/3905)

Reply:
v1.1.8 is now available https://github.com/opencontainers/runc/releases/tag/v1.1.8